slices/libx264-163.yaml

```diff
@@ -0,0 +1,8 @@
+package: libx264-163
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libx264.so.163*:
```

Comment:
I don't think we need the wildcard at the end here, being the only file there is.

Reply:
y